scripts/process-stream.sh

```
@@ -0,0 +1,422 @@
+      > "$SILENCES"
+    log "  Found $(wc -l < "$SILENCES") silence boundaries"
+else
+    log "Pass 1b: Silences file exists, skipping"
+fi
+
+# --- 1c: Volume per 10-second window ---
+VOLUME_FILE="$OUT_DIR/volume-per-10s.txt"
+if [ ! -f "$VOLUME_FILE" ]; then
+    log "Pass 1c: Measuring volume per 10s window..."
+    DURATION=$(ffprobe -v error -show_entries format=duration -of csv=p=0 "$AUDIO" 2>/dev/null | cut -d. -f1)
+    > "$VOLUME_FILE"
```

Shellcheck (0.11.0)

[warning] This redirection doesn't have a command. Move to its command (or use 'true' as no-op).

(SC2188)